### PR TITLE
infra: Add specific feature-environment for hypershift testing 4.16

### PR DESCRIPTION
**Pull request by sshnaidm**

### CODE REVIEW

File: feature-configs/hypershift-ci/container-mount-namespace/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../deploy/container-mount-namespace/
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/container-mount-namespace/kustomization.yaml:4]: please check I think this use a MCP that is not supported

File: feature-configs/hypershift-ci/fec/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../deploy/fec
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/fec/kustomization.yaml:5]: lets remove or put this folder in a comment that FEC was not tested for hypershift

File: feature-configs/hypershift-ci/gatekeeper/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/gatekeeper/kustomization.yaml:1]: we don't need this. gatekeeper is not supported by us any more

File: feature-configs/hypershift-ci/localhost-workaround/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../deploy/localhost-workaround
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/localhost-workaround/kustomization.yaml:5]: please check this file I think it use a MCP

File: feature-configs/hypershift-ci/n3000/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/n3000/kustomization.yaml:1]: remove this one not supported

File: feature-configs/hypershift-ci/sctp/kustomization.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/sctp/kustomization.yaml:1]: this one will not work as no MCP

File: feature-configs/hypershift-ci/sro/kustomization.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/sro/kustomization.yaml:1]: no need we don't support this one

File: feature-configs/hypershift-ci/sro/nfd-subscription.yaml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/sro/nfd-subscription.yaml:1]: no need any more

File: feature-configs/hypershift-ci/sro/sro-subscription.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (SchSeba) [inline feature-configs/hypershift-ci/sro/sro-subscription.yaml:1]: no need any more

File: feature-configs/hypershift-ci/sctp/kustomization.yaml (hunk 6 above)
Comment (sshnaidm) [reply feature-configs/hypershift-ci/sctp/kustomization.yaml:1]: should I remove it?

File: feature-configs/hypershift-ci/localhost-workaround/kustomization.yaml (hunk 4 above)
Comment (sshnaidm) [reply feature-configs/hypershift-ci/localhost-workaround/kustomization.yaml:5]: yep, removing

File: feature-configs/hypershift-ci/sctp/kustomization.yaml (hunk 6 above)
Comment (sshnaidm) [reply feature-configs/hypershift-ci/sctp/kustomization.yaml:1]: ok, removed sctp

File: feature-configs/hypershift-ci/container-mount-namespace/kustomization.yaml (hunk 1 above)
Comment (sshnaidm) [reply feature-configs/hypershift-ci/container-mount-namespace/kustomization.yaml:4]: yes, removing

**Comment by sshnaidm** [comment]
results:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/53982/rehearse-53982-periodic-ci-openshift-release-master-nightly-4.16-e2e-telco5g-hcp-cnftests/1808781670589927424/artifacts/e2e-telco5g-hcp-cnftests/telco5g-cnf-tests/artifacts/cnftests.html

**Comment by sshnaidm** [comment]
Done in https://github.com/openshift-kni/cnf-features-deploy/pull/1959

**Pull request closed without merging**